app/views/users/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>Hello Rails</h1>
+
+<%= form_with model: @user do |form| %>
+  <% if @user.errors.any? %>
+      <div id="error_explanation">
+        <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+        <ul>
+        <% @user.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+        </ul>
+      </div>
+    <% end %>
```

Comment:
**Optional:**

- [ ] Kindly also move the logic to a helper. The reason, as already highlighted above.